### PR TITLE
📓 Updated hungarian

### PR DESCRIPTION
**Pull request by KristofKekesi**

### CODE REVIEW

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 1]
```diff
@@ -305,7 +305,7 @@
         </trans-unit>
         <trans-unit id="BaseLayoutContextFlyoutSortBy.Text" translate="yes" xml:space="preserve">
           <source>Sort by</source>
-          <target state="translated">Rendezés</target>
+          <target state="translated">Csoportosítás</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:308]: Sort by = Rendezés

```suggestion
          <target state="translated">Rendezés</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 2]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="NavToolbarArrangementOption_Type.Content" translate="yes" xml:space="preserve">
           <source>Type</source>
-          <target state="new">Type</target>
+          <target state="translated">Típus</target>
         </trans-unit>
         <trans-unit id="NavToolbarSortByRadioButtons.Header" translate="yes" xml:space="preserve">
           <source>Sort By</source>
-          <target state="new">Sort By</target>
+          <target state="translated">Csoportosítás</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2822]:
```suggestion
          <target state="translated">Rendezés</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 3]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="NavToolbarSortDirectionRadioButtons.Header" translate="yes" xml:space="preserve">
           <source>Sort direction</source>
-          <target state="new">Sort direction</target>
+          <target state="translated">Csoportosítás iránya</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2826]:
```suggestion
          <target state="translated">Rendezés iránya</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 4]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="ItemTimeText_LastWeek" translate="yes" xml:space="preserve">
           <source>Last week</source>
-          <target state="new">Last week</target>
+          <target state="translated">Előző héten</target>
         </trans-unit>
         <trans-unit id="ItemTimeText_ThisMonth" translate="yes" xml:space="preserve">
           <source>This month</source>
-          <target state="new">This month</target>
+          <target state="translated">Ezen a hónapon</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2878]:
```suggestion
          <target state="translated">Ebben a hónapban</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 5]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="ItemTimeText_LastMonth" translate="yes" xml:space="preserve">
           <source>Last month</source>
-          <target state="new">Last month</target>
+          <target state="translated">Előző hónapon</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2882]:
```suggestion
          <target state="translated">Előző hónapban</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 6]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="ItemTimeText_ThisYear" translate="yes" xml:space="preserve">
           <source>This year</source>
-          <target state="new">This year</target>
+          <target state="translated">Idén</target>
         </trans-unit>
         <trans-unit id="ItemTimeText_Older" translate="yes" xml:space="preserve">
           <source>Older</source>
-          <target state="new">Older</target>
+          <target state="translated">Később</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2890]:
```suggestion
          <target state="translated">Régebben</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 7]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="SettingsAboutPrivacyPolicy.Text" translate="yes" xml:space="preserve">
           <source>Privacy policy</source>
-          <target state="new">Privacy policy</target>
+          <target state="translated">Adatvédelmi irányelvek</target>
         </trans-unit>
         <trans-unit id="SettingsAboutPrivacyPolicyDescription.Text" translate="yes" xml:space="preserve">
           <source>View Files privacy policy</source>
-          <target state="new">View Files privacy policy</target>
+          <target state="translated">A Files program adatvédelmi irányelve</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2914]:
```suggestion
          <target state="translated">A Files program adatvédelmi irányelvei</target>
```

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 8]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="SettingsAboutPrivacyPolicyListViewItem.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>View Files privacy policy</source>
-          <target state="new">View Files privacy policy</target>
+          <target state="translated">A Files program adatvédelmi irányelve</target>
```
Comment (nvi9) [inline Files/MultilingualResources/Files.hu-HU.xlf:2918]:
```suggestion
          <target state="translated">A Files program adatvédelmi irányelvei</target>
```

**Review by nvi9 — CHANGES_REQUESTED** [review]

**Comment by KristofKekesi** [comment]
Thanks @nvi9.

File: Files/MultilingualResources/Files.hu-HU.xlf (hunk 7 above)
Comment (nvi9) [reply Files/MultilingualResources/Files.hu-HU.xlf:2914]: @KristofKekesi don't forget about this one too 👀

File: Files/MultilingualResources/Files.hu-HU.xlf [hunk 9]
```diff
@@ -2775,153 +2775,153 @@
...
         </trans-unit>
         <trans-unit id="NavToolbarGroupByOption_None.Content" translate="yes" xml:space="preserve">
           <source>None</source>
-          <target state="new">None</target>
+          <target state="translated">Nincs</target>
         </trans-unit>
         <trans-unit id="NavToolbarArrangementOption_Name.Content" translate="yes" xml:space="preserve">
           <source>Name</source>
-          <target state="new">Name</target>
+          <target state="transalted">Név</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.hu-HU.xlf:2794]:
```suggestion
          <target state="translated">Név</target>
```

**Review by tsvietOK — APPROVED** [review]

**Pull request merged**